support-frontend/app/services/AuthenticationService.scala

```diff
@@ -27,45 +28,37 @@ object AccessCredentials {
   }
   case class Token(tokenText: String) extends AccessCredentials
 }
-case class IdMinimalUser(id: String, displayName: Option[String])
-case class AuthenticatedIdUser(credentials: AccessCredentials, minimalUser: IdMinimalUser)
 
 class AsyncAuthenticationService(identityPlayAuthService: IdentityPlayAuthService)(implicit ec: ExecutionContext) {
 
   import AsyncAuthenticationService._
 
-  def authenticateUser(requestHeader: RequestHeader): Future[AuthenticatedIdUser] =
+  def tryAuthenticateUser(requestHeader: RequestHeader): Future[Option[User]] =
     identityPlayAuthService.getUserFromRequest(requestHeader)
-      .map { case (credentials, user) => buildAuthenticatedUser(credentials, user) }
-      .unsafeToFuture()
-
-  def tryAuthenticateUser(requestHeader: RequestHeader): Future[Option[AuthenticatedIdUser]] =
-    authenticateUser(requestHeader)
-      .map(user => Option(user))
-      .handleError { err =>
-        logUserAuthenticationError(err)
-        None
+          .map { case (_, user) => user }
```

Comment:
we used to need the credential object, but this is not used any more, so I flattened out the AuthenticatedIdUser object completely